Doxyfile

```diff
@@ -894,8 +894,8 @@ FILE_PATTERNS          = *.c \
                          *.ucf \
                          *.qsf \
                          *.as \
-                         *.js \
-                         Makefile
```

Comment:
this lets us setup doxygen (in this Doxyfile) to treat warnings as errors, which would provide a better report from the build_doxygen CI workflow

Reply:
done. doxygen builds without warnings nor errors